dashboard/lib/tasks/setup_or_migrate.rake

```diff
@@ -3,8 +3,8 @@ require 'active_record/errors'
 namespace :db do
   def database_exists?
     Rake::Task['environment'].invoke
-    ActiveRecord::Base.connection
-  rescue ActiveRecord::NoDatabaseError
+    ActiveRecord::Base.connection.execute('SHOW TABLES')
+  rescue ActiveRecord::NoDatabaseError, ActiveRecord::StatementInvalid
```

Comment:
I think these changes were needed because ProxySQL returned slightly different error messages in the case of missing databases, probably because of the different behavior through its db-connection schema cache.